python/tutorial.py

```diff
@@ -99,11 +99,16 @@
 # >>> proposal
 # {'script': {'text': 'return Calls:call("open_network")'}}
 
-member_client = ccf.clients.CCFClient(host, port, ca, member_cert, member_key)
+member_client = ccf.clients.CCFClient(
```

Comment:
Does this mean that once a client has been created, all requests sent will be signed or not, depending on the value passed to the `signing_auth` argument here? This seems a little restricting, especially that these are now set at the granularity of a handler. I prefer the current way where `signed` is set per request. Of course, we can still set a `signing_auth` in the constructor of `CCFClient` so that `signing_auth` can be different from `session_auth`.

Reply:
Yes, I mentioned this in the description. I do not believe this is restricting, it's never used at the moment.